host/sgx/create.c

```diff
@@ -991,6 +994,10 @@ oe_result_t oe_sgx_build_enclave(
         /* Only opt into the feature if CPU (SGX2) supports the MISC region. */
         if (_is_misc_region_supported())
             context->capture_pf_gp_exceptions_enabled = 1;
+#if !defined(OEHOSTMR)
+        else /* Enable #PF simulation (debug-mode only) */
+            oe_host_enable_debug_pf_simulation();
```

Comment:
Does this function need to be marked SGX only? E.g: oe_sgx_host_enable_pf_simulation